web/modules/custom/asset_distribution/config/install/field.field.rdf_entity.asset_distribution.field_ad_licence.yml

```diff
@@ -9,7 +9,7 @@ field_name: field_ad_licence
 entity_type: rdf_entity
 bundle: asset_distribution
 label: License
-description: ''
+description: 'If you cannot find the desired license in the list above, please request a new one by <a href="/contact">contacting us</a>.'
```

Comment:
This hardcoded path (`/contact`) will not work on sites where Drupal is installed in a sub-directory. As Joinup is distro, we cannot predict if such cases will exist or not. So, we need to provide the full-qualified. Fortunately the field API allows tokens. I think `[site:url]` is OK to be used as prefix to "contact". Also, let's click on the link, in the Behat test, and check if we're redirected to Contact page.